example/Makefile

```diff
@@ -37,17 +52,20 @@ run:
 	fi
 	$(KUBECTL) apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MANAGER_VERSION)/cert-manager.crds.yaml
 	$(KUBECTL) create namespace topolvm-system
-	$(KUBECTL) label namespace topolvm-system topolvm.cybozu.com/webhook=ignore
-	$(KUBECTL) label namespace kube-system topolvm.cybozu.com/webhook=ignore
+	$(KUBECTL) label namespace topolvm-system $(LABEL_KEY)/webhook=ignore
```

Comment:
`topolvm.io` and `topolvm.cybozu.com` themselves are not label keys.
So how about changing the variable name from `LABEL_KEY` to `DOMAIN_NAME`?

Reply:
fixed.